app.py

```
@@ -1,61 +1,405 @@
+    return jsonify({
+        "status": "healthy",
+        "service": "source2social",
+        "version": "1.0.0"
+    })
 
 if __name__ == "__main__":
-    # Development server
-    app.run(debug=True, host="0.0.0.0", port=5001)
+    port = int(os.getenv("PORT", 5000))
+    debug = os.getenv("DEBUG", "True").lower() == "true"
+    app.run(debug=debug, host="0.0.0.0", port=port)
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)